release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2720,6 +2720,78 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+[id="ocp-4-15-12"]
+=== RHSA-2024:2664 - {product-title} 4.15.12 bug fix and security update
+
+Issued: 2024-05-09
+
+{product-title} release 4.15.12, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:2664[RHSA-2024:2664] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:2669[RHSA-2024:2669] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.12 --pullspecs
+----
+
+[id="ocp-4-15-12-enhancements"]
+==== Enhancements
+
+The following enhancements are included in this z-stream release:
+
+[id="ocp-4-15-12-apiversion-upgrade-v1beta1"]
+===== API Version for CRD ClusterTriggerBinding, TriggerTemplate, and EventListener upgraded from v1alpha1 to v1beta1
+
+* Previously, the API version for CRD ClusterTriggerBinding, TriggerTemplate, and EventListener was v1alpha1. With this release, the API version is upgraded to v1beta1 so that the pipelines plugin supports the latest Pipeline Trigger API version for CRD ClusterTriggerBinding, TriggerTemplate, and EventListener. (link:https://issues.redhat.com/browse/OCPBUGS-31445[*OCPBUGS-31445*])
+
+[id="ocp-4-15-12-pipeline-list-page-improvement"]
+===== PipelineRun list view performance improvement
+
+* Previously, in the PipelineRun list page, all of the TaskRuns were fetched and separated based on their PipelineRun name. With this release, TaskRuns are only fetched for failed and cancelled PipelineRuns, and a caching mechanism is added to fetch the PipelineRuns and TaskRuns that are associated with the failed and cancelled PipelineRuns. (link:https://issues.redhat.com/browse/OCPBUGS-31799[*OCPBUGS-31799*])
+
+[id="ocp-4-15-12-installer-handling-new-character"]
+===== Installer handles the escaping of the "%" character
+
+* Previously, if a cluster was installed using proxy and the proxy information contained escaped characters in the format "%XX", the installation failed. With this release, the Installer now handles the escaping of the "%"" character. (link:https://issues.redhat.com/browse/OCPBUGS-32259[*OCPBUGS-32259*])
```

Comment:
Not sure about double-quotes for character/syntax usage (also I think there's an extra `"` in the very last instance). Suggestion:
```suggestion
===== Installer handles the escaping of the % character

* Previously, if a cluster was installed using a proxy, and the proxy information contained escaped characters in the format `%XX`, the installation failed. With this release, the installer now handles the escaping of the `%` character. (link:https://issues.redhat.com/browse/OCPBUGS-32259[*OCPBUGS-32259*])
```
^ Note lowercase "installer"; I think we possibly only init-cap it for "Assisted Installer"? Just going off of https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#assisted-installer and SSG.